src/entry-server.js

```diff
@@ -59,21 +57,19 @@ export async function renderError({ err, req, res, buildManifest }) {
         throw err;
     }
 
-    // Render page that goes to into <div id="root"></div>
-    const createElement = (Component, props) => <Component { ...props } serverContext={ { err, req, res } } />;
-    const rootHtml = renderToString(
+    res.write(renderInitialBlock({ helmet: Helmet.renderStatic(), buildManifest }));
+
+    // Render HTML that goes to into <div id="root"></div>
+    const createElement = (Component, props) => <Component { ...props } serverContext={ { req, res } } />;
+    const streamedRootHtml = renderToNodeStream(
```

Comment:
What about `pageContentStream` instead?

Reply:
+1 i like it 

@satazor

Reply:
It should be `rootContentsStream` or `rootHtmlStream`. I like having `root` in the name because there's a direct correlation between the `#root` and the stream that renders it.

Reply:
+1 for rootHtmlStream

Reply:
When I see/read `root` I think "this represents the topmost level possible", and that's not what this var represents here, hence my suggestion. Maybe the issue lies in the div's ID. Should it be something like `main` or `app`?

Anyway, this is not blocking by any means. Feel free to ignore these comments.

Reply:
Perhaps we could change `#root` to something else? I used `#root` because thats what most react tutorials (and I guess the official react documentation) uses.

Reply:
yep, its in the docs and projects like cra and next use it.

`We call this a “root” DOM node because everything inside it will be managed by React DOM.`

From a React DOM perspective its right.

Reply:
Yeah, that makes sense for ReactDOM. But in the context of this file, it is not the topmost level. Not to me anyway :)

Lets move on, please, this is not a big deal.